app/reducers/treatment.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios'
+import { browserHistory } from 'react-router'
+import { isBrowser } from '../utils'
+
+const initialState = {
```

Comment:
Hmm. I do recall that, but I have seen others ([eg](https://medium.com/@kylpo/redux-best-practices-eef55a20cc72#.aih1zjzi4)) with different opinions I can go either way, but here is the reason I like an empty object:

With an initialized dummy object, we are subject to treating this like any other `treatment` that we render into interested views. However, an empty object simply tells our view that it does not currently exist. This forgoes brittle checks like` id !== 0` or some other way of identifying that the initial `treatment` is not real. Happy to hear an argument where this might be preferred, tho.